### ISSUE TITLE
Using `@ import "compass"` does not include all mixins

### ISSUE THREAD

**Issue by MoOx** [Compass/compass#957]
Layout stuffs are not included when using  `@ import "compass"`. Maybe other things are missing too.

**Comment by mckramer** [comment]
Not sure if you figured this one out, but it is the expected functionality. See [Compass Reference](http://compass-style.org/reference/compass/).

**Comment by MoOx** [comment]
I've seen that. But I ask @chriseppstein & he answer me to open a issue :)
I don't get the point of having a  `@ import "compass"`  shortcut which do not import all things under `compass/*`
I found weird to code this

``` scss
@ import "compass"
@ import "compass/layout"
```

**Comment by silvenon** [comment]
He has a point. To meet this halfway, maybe there should be `compass/all` import?

**Comment by mirisuzanne** [comment]
I agree, modules should only be left out when they have automatic output (as per reset).
